adbms/src/test/scala/de/up/hpi/informationsystems/adbms/definition/RecordTest.scala

```diff
@@ -60,29 +60,29 @@ class RecordTest extends WordSpec with Matchers {
       }
 
       "project to itself, when projected by an empty list" in {
-        emptyRecord.project(Seq.empty) should equal(Success(emptyRecord))
+        emptyRecord.project(Set.empty) should equal(Success(emptyRecord))
       }
 
       "not allow projection, when projecting by any column definition" in {
-        emptyRecord.project(Seq(ColumnDef[Any](""))).isFailure shouldBe true
+        emptyRecord.project(Set(ColumnDef[Any](""))).isFailure shouldBe true
         emptyRecord.project(new RowRelation {
-          override def columns: Seq[UntypedColumnDef] = Seq(ColumnDef[Any](""))
-        }).isFailure shouldBe true
+          override def columns: Set[UntypedColumnDef] = Set(ColumnDef[Any](""))
+        }.columns).isFailure shouldBe true
```

Comment:
this test can be removed as we do not have this overload (`project(new Relation{})`) any more